.github/workflows/build-rock.yaml

```diff
@@ -31,7 +31,9 @@ jobs:
           fi
 
       - name: Upload rock
-        uses: actions/upload-artifact@v4
+        # NOTE: using v3 (for this and other actions) due to node versions
+        # on self-hosted runners (no node20)
```

Comment:
Do we want to disable Renovate from updating these actions?

Example: https://github.com/canonical/craft-providers/blob/b42e6c8211578c9d7e74e9486d4b9bb38f1a8c3f/.github/renovate.json5#L4

Reply:
(Not a blocker for this PR, just a nice to have so we don't get Renovate updating the actions and breaking CI)

Reply:
I think so
do I just add strings like `"actions/upload-artifact@v4"` to `ignoreDeps`?